### PR TITLE
mengheng fourth commit of the day

### PR DESCRIPTION
**Pull request by Meng-Heng**

### CODE REVIEW

File: developer/language/reference/reset.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+`storeName`
+:   The name of the variable store to be reset to its original value.
+
+## Description
+
+The `reset()` statement is used on the right hand (output) side of a
+rule to restore a variable to the current default value - either the
+store content as defined in the keyboard source file, or the value
+persisted previously in the system settings by [`save()`](save.php) or
```
Comment (jahorton) [inline developer/language/reference/reset.md:26]:
```suggestion
persisted previously in the system settings by [`save()`](save) or
```

File: developer/language/reference/save.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Keyman for macOS 11.0.221 added support for `if()`, `set()`, `save()`
+and `reset()` for variable stores.
+
+## See also
+
+-   [Using variable stores](../guide/variable-stores)
+-   [if() statement](./if)
+-   [reset() statement](./reset) 
+-   [set() statement](./set)
+-   [store() statement](./store)
```
Comment (jahorton) [inline developer/language/reference/save.md:60]:
```suggestion
-   [`if()` statement](./if)
-   [`reset()` statement](./reset) 
-   [`set()` statement](./set)
-   [`store()` statement](./store)
```

File: developer/language/reference/reset.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Keyman for macOS 11.0.221 added support for `if()`, `set()`, `save()`
+and `reset()` for variable stores.
+
+## See also
+
+-   [Using variable stores](../guide/variable-stores)
+-   [if() statement](./if)
+-   [save() statement](./save)
+-   [set() statement](./set)
+-   [store() statement](./store)
```
Comment (jahorton) [inline developer/language/reference/reset.md:59]:
```suggestion
-   [`if()` statement](./if)
-   [`save()` statement](./save)
-   [`set()` statement](./set)
-   [`store()` statement](./store)
```

**Review by jahorton — COMMENTED** [review]
Just happened to catch some code styling in links that the Pandoc-based converter itself didn't handle properly.

**Review by Meng-Heng — COMMENTED** [review]
done reviewing

**Review by jahorton — APPROVED** [review]

**Pull request merged**